iceoryx_utils/include/iceoryx_utils/error_handling/error_handling.hpp

```diff
@@ -51,6 +51,7 @@ namespace iox
     error(POSH__RUNTIME_ROUDI_GET_MW_APPLICATION_WRONG_IPC_MESSAGE_RESPONSE) \
     error(POSH__RUNTIME_ROUDI_CONDITION_VARIABLE_CREATION_UNDEFINED_BEHAVIOR) \
     error(POSH__RUNTIME_ROUDI_EVENT_VARIABLE_CREATION_UNDEFINED_BEHAVIOR) \
+    error(POSH__RUNTIME_APP_WITH_SAME_NAME_STILL_RUNNING) \
```

Comment:
Please remove this - we explicitly want that! What we do not want that two applications (with a random name initialize a runtime with the same name)

Reply:
Added to #611